packages/remix/src/executors/serve/serve.impl.ts

```diff
@@ -0,0 +1,93 @@
+import { workspaceRoot, type ExecutorContext } from '@nx/devkit';
+import { createAsyncIterable } from '@nx/devkit/src/utils/async-iterable';
+import { waitForPortOpen } from '@nx/web/src/utils/wait-for-port-open';
+import { fork } from 'node:child_process';
+import { join } from 'node:path';
+import { type RemixServeSchema } from './schema';
+
+function normalizeOptions(schema: RemixServeSchema) {
+  return {
+    ...schema,
+    port: schema.port ?? 4200,
+    debug: schema.debug ?? false,
+    manual: schema.manual ?? false,
+  } as RemixServeSchema;
+}
+
+function buildRemixDevArgs(options: RemixServeSchema) {
+  const args = [];
+
+  if (options.command) {
+    args.push(`--command=${options.command}`);
+  } else {
+    args.push(`--port=${options.port}`);
+  }
```

Comment:
With Ocean, adding `port` arg when there's a `command` is causing an `address already in use` error, so using the `if/else` here to only add it if a custom command is not being used to serve the app.

`port` will still be added to env vars so that the custom command can still get access to the port option